src/lib/operators/materialize.cpp

```diff
@@ -0,0 +1,71 @@
+#include "materialize.hpp"
+
+#include "resolve_type.hpp"
+#include "storage/base_column.hpp"
+#include "storage/chunk.hpp"
+#include "storage/create_iterable_from_column.hpp"
+#include "storage/table.hpp"
+
+namespace opossum {
+
+Materialize::Materialize(const std::shared_ptr<const AbstractOperator> in)
+    : AbstractReadOnlyOperator{in} {}
+
+const std::string Materialize::name() const { return "Materialize"; }
+const std::string Materialize::description(DescriptionMode description_mode) const { return name(); }
```

Comment:
That's the default, isn't it? No need to have this function here then. Also, returning const values doesn't make sense and we abolished that ages ago.

Reply:
This is in fact how these two methods are declared in `AbstractOperator`.